bin/node/runtime/src/voter_bags.rs

```diff
@@ -233,3 +233,206 @@ pub const THRESHOLDS: [u64; 200] = [
 	17_356_326_621_502_140_416,
 	18_446_744_073_709_551_615,
 ];
+
+pub const THRESHOLDS_BALANCE: [crate::Balance; 200] = [
```

Comment:
exact dupe of above but Balance type? Would be nice if we could just have one list and go between types with zero cost